src/main/rx/cc2500_frsky_x.c

```diff
@@ -343,15 +345,11 @@ rx_spi_received_e frSkyXHandlePacket(uint8_t * const packet, uint8_t * const pro
     case STATE_DATA:
         if (cc2500getGdo() && (frameReceived == false)){
             uint8_t ccLen = cc2500ReadReg(CC2500_3B_RXBYTES | CC2500_READ_BURST) & 0x7F;
-            ccLen = cc2500ReadReg(CC2500_3B_RXBYTES | CC2500_READ_BURST) & 0x7F; // read 2 times to avoid reading errors
```

Comment:
Yes.
It was there, because of how cc2500 handles GDO0 interrupt - it is configured so that the pin will go high once a certain treshold is reached in the rx fifo. Sometimes readings were incorrect, so someone just put a second read there as a safety measure (maybe sometimes it triggered earlier than it should and the fifo did not contain the final byte yet). In my refactored code it will read it continuously until it reaches the correct amount of data in the fifo - 32 bytes for frsky x and 35 for frsky x eu lbt.